src/storage/src/hummock/compactor/mod.rs

```diff
@@ -677,9 +678,24 @@ pub fn start_shared_compactor(
                                         .report_compaction_task(report_compaction_task_request)
                                         .await
                                     {
-                                        Ok(_) => {}
+                                        Ok(_) => {
+                                            // TODO: remove this method after we have running risingwave cluster with fast compact algorithm stably for a long time.
+
+                                            if context.storage_opts.check_fast_compaction_result
```

Comment:
Let's rename `check_fast_compaction_result` to `check_compaction_result`

Reply:
+1

Reply:
fixed